Code/Engine/Foundation/IO/FileSystem/DeferredFileWriter.h

```diff
@@ -22,7 +22,7 @@ class EZ_FOUNDATION_DLL ezDeferredFileWriter : public ezStreamWriter
 
   /// \brief Upon calling this the content is written to the file specified with SetOutput().
   /// The return value is EZ_FAILURE if the file could not be opened or not completely written.
-  ezResult Close(); // [tested]
+  ezResult Close(bool* out_bWasWrittenTo = nullptr); // [tested]
```

Comment:
Isn't out_bWasWrittenTo the same as the returned ezResult?

Reply:
No, if the content is identical, the result will be true, but written to will be false.